src/main/java/com/jensjansson/ce/views/persons/EmployeesView.java

```diff
@@ -98,6 +104,7 @@ public void deletePerson() {
                 .setFlexGrow(1);
         grid.addColumn(new ComponentRenderer<>(this::createContactInfo))
                 .setFlexGrow(2);
+        grid.addColumn(new ComponentRenderer<>(this::createPresentComponent)).setFlexGrow(1);
```

Comment:
Could be written as `PresenceComponent::new` without the helper method, as long as `PresenceComponent` is changed to be `static` (which it should be).

Reply:
Thanks for reviewing it. I tested and it works without making it static. If do make it static, I'll have to pass the `localUser` and possibly the `topicId` also in the constructor. Should I still change the class to static?

Reply:
Oh. I missed that the constructor was implicitly accessing the enclosing component instance which thus means that `static` won't work without further changes. `getTopicId` could be changed to also be `static`, but the `localUser` instance is more problematic.

My personal preference is to be careful with such implicit dependencies to the enclosing class in things that might be useful to have as a standalone thing, but that's just a personal preference...

Reply:
I pulled the class out of EmployeesView